crates/agglayer-config/tests/snapshots/backup__backup_enabled.snap

```diff
@@ -0,0 +1,68 @@
+---
+source: crates/agglayer-config/tests/backup.rs
+expression: config
+---
+prover-entrypoint = "http://127.0.0.1:8080"
+
+[full-node-rpcs]
+
+[l2]
+rpc-timeout = "45s"
+
+[proof-signers]
+
+[log]
+level = "info"
+outputs = []
+format = "pretty"
+
+[rpc]
+port = 9090
+host = "0.0.0.0"
+request-timeout = "3m"
+
+[rate-limiting]
+send-tx = "unlimited"
+
+[rate-limiting.network]
+
+[outbound.rpc.settle]
+max-retries = 3
+retry-interval = "7s"
+confirmations = 1
+settlement-timeout = "20m"
+
+[l1]
+chain-id = 1337
+node-url = "http://zkevm-mock-l1-network:8545/"
+ws-node-url = "ws://zkevm-mock-l1-network:8546/"
+max-reconnection-elapsed-time = "10s"
+rollup-manager-contract = "0xb7f8bc63bbcad18155201308c8f3540b07f84f5e"
+polygon-zkevm-global-exit-root-v2-contract = "0xb7f8bc63bbcad18155201308c8f3540b07f84f5e"
+rpc-timeout = "45s"
+
+[auth.local]
+private-keys = []
+
+[telemetry]
+prometheus-addr = "0.0.0.0:3000"
+
+[epoch.block-clock]
+epoch-duration = 6
+genesis-block = 0
+
+[shutdown]
+runtime-timeout = "5s"
+
+[certificate-orchestrator]
+input-backpressure-buffer-size = 1000
+
+[certificate-orchestrator.prover.sp1-local]
+
+[storage]
+db-path = "/tmp/agglayer/tests/fixtures/valide_config/storage"
+
+[storage.backup.enabled]
+path = "./test/"
```

Comment:
Here is what I'm talking about in the other comment. It would become:
```toml
[storage.backup]
path = "./test/"
```

or just:
```toml
storage.backup = "disabled"
```
to disable backups explicitly